### PR TITLE
Enable 'until' attribute in API

### PR DESCRIPTION
**Pull request by ravl1084**

### CODE REVIEW

File: inthe_am/taskmanager/serializers/task.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ class TaskSerializer(serializers.Serializer):
     modified = serializers.DateTimeField(read_only=True)
     start = serializers.DateTimeField(required=False)
     wait = serializers.DateTimeField(required=False)
+    until = serializers.DatetimeField(required=False
```
Comment (coddingtonbear) [inline inthe_am/taskmanager/serializers/task.py]: Looks like you might've missed a `)` at the end of this line.

**Comment by ravl1084** [comment]
Ok, I fixed it :)

File: inthe_am/taskmanager/serializers/task.py [hunk 2]
```diff
@@ -15,6 +15,7 @@ class TaskSerializer(serializers.Serializer):
     modified = serializers.DateTimeField(read_only=True)
     start = serializers.DateTimeField(required=False)
     wait = serializers.DateTimeField(required=False)
+    until = serializers.DatetimeField(required=False)
```
Comment (coddingtonbear) [inline inthe_am/taskmanager/serializers/task.py]: Whoops, missed this when reviewing this; Python is case-sensitive, so this has to be `DateTimeField`. 
 I've manually fixed this in 6305cc8565b253c2d0f0c12230a0a70286d00e65, though, so don't worry about updating the PR.

**Comment by coddingtonbear** [comment]
Just released these changes a moment ago -- let me know if you still have trouble using `until`!

**Pull request merged**